group-coordinator/src/test/java/org/apache/kafka/coordinator/group/consumer/TargetAssignmentBuilderTest.java

```diff
@@ -222,8 +231,25 @@ public void testCreateAssignmentMemberSpec() {
 
         Map<String, TopicMetadata> subscriptionMetadata = new HashMap<String, TopicMetadata>() {
             {
-                put("foo", new TopicMetadata(fooTopicId, "foo", 5));
-                put("bar", new TopicMetadata(barTopicId, "bar", 5));
+                put("foo", new TopicMetadata(fooTopicId, "foo", 5,
+                    mkMap(
+                        mkEntry(0, Collections.emptySet()),
+                        mkEntry(1, Collections.emptySet()),
+                        mkEntry(2, Collections.emptySet()),
+                        mkEntry(3, Collections.emptySet()),
+                        mkEntry(4, Collections.emptySet())
+                    )
```

Comment:
As the racks are empty, we don't have to define the partitions, right? Could we just used `Collections.emptyMap()`? There are other cases in this file.

Reply:
I kept it just in case we wanted to add the rack information